sgrep/tests/go/equivalence_const_propagation.go

```diff
@@ -0,0 +1,10 @@
+package Foo
+
+const Bar = "password"
+
+func foo() {
+   const sql = "%s"
+   const formatted = snprintf(sql, Bar)
+   // TODO
```

Comment:
should rename the .sgrep file .TODO then

Reply:
there is already an equivalence_constant_propagation, so maybe equivalence_constant_dataflow.TODO
or equivalence_constant_const_tracking.TODO

Reply:
ok